src/github.com/matrix-org/dendrite/appservice/storage/storage.go

```diff
@@ -20,6 +20,7 @@ import (
 
 	// Import postgres database driver
 	_ "github.com/lib/pq"
```

Comment:
You are not using this here anymore

Reply:
There are some other appearances where you just removed the import but not the comment for it.
And some some more of this